python/lsst/dax/imgserv/api_soda.py

```diff
@@ -203,8 +207,12 @@ def img_async():
         else:
             soda_url = url_for('api_image_soda.img_async', _external=True)
             return _service_response(soda_url)
+    _params = _getparams()
+    if any(param in ["BAND", "TIME", "POL"] for param in _params):
```

Comment:
I'd probably make a function for this check as it's the same as the one above.

Reply:
Done.